### PR TITLE
Update prime-protocol adapter to use MAINNET v1.6.0 PToken addresses

### PR DESCRIPTION
**Pull request by nbayindirli**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Logo (High resolution, will be shown with rounded borders):
Logo BLUE

##### Current TVL: 
* $946.28k

##### Treasury Addresses (if the protocol has treasury)
* n/a

##### Chain(s):
* MOONBEAM
* AVALANCHE
* ARBITRUM
* ETHEREUM
* More TK

##### Short Description (to be shown on DefiLlama):
* Prime Protocol is the first natively cross-chain decentralized liquidity protocol, serving as a prime brokerage for the entire crypto ecosystem. Borrowers can deposit collateral on any supported blockchain, and receive liquidity on any other chain regardless of original collateral location. All borrows and deposits are cross-margined, so users have the experience of operating from one account across many blockchains.

##### Token address and ticker if any:
* n/a

##### Category (full list at https://defillama.com/categories) *Please choose only one:
* Lending

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
* Chainlink

##### forkedFrom (Does your project originate from another project):
* Not forked

##### methodology (what is being counted as tvl, how is tvl being calculated):
* Summates all Prime Protocol deposits across all networks and markets (borrows not included).

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/prime-protocol exports TVL: 
        
 
 
    arbitrum                  128.09 k
    arbitrum-borrowed         134.09 k
    avax                      15.74 k
    avax-borrowed             5.01 k
    moonbeam                  44.87 k
    moonbeam-borrowed         45.69 k
    ethereum-borrowed         0.08728250000000001
    ethereum                  247.84 k
    borrowed                  184.79 k
    
    total                    436.55 k

File: projects/prime-protocol/index.js [hunk 1]
```diff
@@ -12,60 +12,60 @@ const MASTER_VIEW_CONTRACT = {
 
 const MOONBEAM_MARKETS = {
   GLMR: {
-    pTokenMarketAddress: '0xdC427cDB81E1532747BEDeB8077a4AEcDbfB585e',
+    pTokenMarketAddress: '0x53d5a47bb874eE688acb479676aD133d47CB9B25',
```
Comment (g1nt0ki) [inline projects/prime-protocol/index.js:15]: what if we track both markets? the key for new markets be {TOKEN}_v1_6 ?

**Comment by llamatester** [comment]
The adapter at projects/prime-protocol exports TVL: 
        
 
 
    avax                      122.10 k
    avax-borrowed             6.10 k
    arbitrum                  211.85 k
    arbitrum-borrowed         137.55 k
    moonbeam                  362.35 k
    moonbeam-borrowed         55.58 k
    ethereum                  249.97 k
    ethereum-borrowed         0.0874338
    borrowed                  199.22 k
    
    total                    946.28 k

**Comment by llamatester** [comment]
The adapter at projects/prime-protocol exports TVL: 
        
 
 
    avax                      122.23 k
    ethereum                  250.22 k
    avax-borrowed             6.08 k
    arbitrum                  212.06 k
    moonbeam                  362.45 k
    ethereum-borrowed         0.08751020000000001
    moonbeam-borrowed         55.58 k
    arbitrum-borrowed         137.69 k
    borrowed                  199.34 k
    
    total                    946.96 k

File: projects/prime-protocol/index.js (hunk 1 above)
Comment (nbayindirli) [reply projects/prime-protocol/index.js:15]: Updates to track current and past markets

**Pull request merged**